core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -103,6 +104,7 @@ java.util.GregorianCalendar
 java.util.HashMap
 java.util.HashSet
 java.util.Hashtable
+java.util.ImmutableCollections$List12
```

Comment:
Just saw https://github.com/jenkinsci/jenkins/pull/6700#issuecomment-1169217726. For reference, I think this is so that tests like https://github.com/jenkinsci/jenkins/blob/a6ec1b024a9b6136677a3cb7505fdb690001bbac/test/src/test/java/hudson/model/ParametersAction2Test.java#L131 could still work. We would not expect such entries to be used in production code, any more than we would older entries like https://github.com/jenkinsci/jenkins/blob/a6ec1b024a9b6136677a3cb7505fdb690001bbac/core/src/main/resources/jenkins/security/whitelisted-classes.txt#L78